### PR TITLE
Fix the version selection to be the latest release

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Switched from checking the committed branch to the draft flag.

## QA
- Open the homepage
- Scroll down to the download button
- Check the Windows and Mac links are to v7.0 

## Details
Fixes https://github.com/canonical-web-and-design/multipass.run/issues/38

### CODE REVIEW

File: js/installversion.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ function getAssetInfo(os) {
     var assets = release.assets;
     for (var q = 0; q < assets.length; q++) {
       var asset = assets[q];
-      if (asset.name.includes(os) && release.target_commitish == 'master') {
+      if (asset.name.includes(os) && !release.draft) {
```
Comment (Saviq) [inline js/installversion.js]: Is there really no "pre-release" bit here? I don't think you'll ever see a draft here as those are only visible to authenticated team members.

Comment (Saviq) [reply js/installversion.js]: https://developer.github.com/v3/repos/releases/#response says there's a `prerelease` boolean? To be sure we could filter on both?

**Comment by anthonydillon** [comment]
@Saviq your right, updated the filter.

**Comment by Saviq** [comment]
Hey, this is still not deployed as far I can see?

**Pull request merged**